cmd/libs/go2idl/go-to-protobuf/protobuf/cmd.go

```diff
@@ -89,6 +89,9 @@ func New() *Generator {
 			`k8s.io/kubernetes/pkg/apis/settings/v1alpha1`,
 			`k8s.io/kubernetes/pkg/apis/storage/v1beta1`,
 			`k8s.io/kubernetes/pkg/apis/storage/v1`,
+			`-k8s.io/client-go/pkg/api/v1`,
```

Comment:
No, `-` means "consider types from this package, but don't generate IDL and Go code"